packages/circuit-ui/components/Table/Table.stories.tsx

```diff
@@ -20,6 +20,7 @@ import Badge from '../Badge';
 import docs from './Table.docs.mdx';
 import { TableProps } from './Table';
 import { Direction } from './types';
+import { getSortDirection } from './utils';
```

Comment:
I don't see this being used in this file. Is this a left-over, unused import?

Reply:
yes that's correct. i will remove it.